crates/matrix-sdk-test/src/test_json/events.rs

```diff
@@ -1,24 +1,22 @@
-use lazy_static::lazy_static;
+use once_cell::sync::Lazy;
 use serde_json::{json, Value as JsonValue};
 
-lazy_static! {
-    pub static ref ALIAS: JsonValue = json!({
-        "content": {
-            "alias": "#tutorial:localhost"
-        },
-        "event_id": "$15139375513VdeRF:localhost",
-        "origin_server_ts": 151393755,
-        "sender": "@example:localhost",
-        "state_key": "",
-        "type": "m.room.canonical_alias",
-        "unsigned": {
-            "age": 703422
-        }
-    });
-}
+pub static ALIAS: Lazy<JsonValue> = Lazy::new(|| {
+    json!({"content": {
```

Comment:
Please put the `"content": {` part on its own line like before and clean up the indentation.